src/core/components/radio/Radio.tsx

```diff
@@ -0,0 +1,131 @@
+import React, { ReactNode, InputHTMLAttributes } from 'react';
+import {
+	label,
+	labelWithSupportingText,
+	radio,
+	labelText,
+	labelTextWithSupportingText,
+	supportingText,
+} from './styles';
+import { Props } from '@guardian/src-helpers';
+
+const LabelText = ({
+	hasSupportingText,
+	children,
+}: {
+	hasSupportingText?: boolean;
+	children: ReactNode;
+}) => {
+	return (
+		<div
+			css={(theme) => [
+				hasSupportingText ? labelTextWithSupportingText : '',
+				labelText(theme.radio && theme),
+			]}
+			className="src-radio-label-text"
+		>
+			{children}
+		</div>
+	);
+};
+
+const SupportingText = ({ children }: { children: ReactNode }) => {
+	return (
+		<div css={(theme) => supportingText(theme.radio && theme)}>
+			{children}
+		</div>
+	);
+};
+
+export interface RadioProps
+	extends InputHTMLAttributes<HTMLInputElement>,
+		Props {
+	/**
+	 * TODO: Add a description or remove as value is inherited from `InputHTMLAttributes<HTMLInputElement>` (as `string | number | readonly string[] | undefined`)
+	 */
+	value: string;
+	/**
+	 * Whether radio button is checked. This is necessary when using the [controlled approach](https://reactjs.org/docs/forms.html#controlled-components) to form state management.
+	 *
+	 * _Note: if you pass the `checked` prop, you MUST also pass an `onChange` handler, or the field will be rendered as read-only._
+	 */
+	checked?: boolean;
+	/**
+	 * TODO: Add a description
+	 */
+	defaultChecked?: boolean;
+	/**
+	 * Appears to the right of the radio button. If a visible label is undesirable (e.g. for layout reasons) use `aria-label` instead.
+	 *
+	 * If label is omitted, supporting text will not appear either.
+	 */
+	label?: ReactNode; // TODO: Should this be `String | ReactNode`?
+	/**
+	 * Additional text or a component that appears below the label
+	 */
+	supporting?: ReactNode; // TODO: Should this be `String | ReactNode`?
```

Comment:
Having clicked through the ReactNode type, `string` is a sub type so maybe this is better than saying `string | ReactNode` but equally maybe it's clearer to users that you can just pass a string?

Reply:
I think it's non-obvious that `ReactNode` includes `string`. I've gone around the houses on this many times. I find `string | ReactNode` more descriptive and useful. It might not be technically adding new information, but from most people's perspective, exposing hidden information is effectively the same as adding new information.

Reply:
Done in https://github.com/guardian/source/pull/902/commits/99a237c097d4d5c0bd791f0dcb927396afd4f558